src/components/Todos.tsx

```diff
@@ -0,0 +1,231 @@
+import React, { useEffect, useState } from "react";
+import "bootstrap/dist/css/bootstrap.css";
+import "./Todos.css";
+import { useNavigate } from "react-router-dom";
+import { ArrowUp, ArrowDown } from "lucide-react";
+import useFetch from "./useFetch";
+
+export interface Todo {
+  id: number;
+  title: string;
+  date: string;
+  isCompleted: boolean;
+}
+
+export default function Todos() {
+  let { todos, loading, error,refetchData } = useFetch("http://localhost:8000/todos");
+  const [todolist, setTodolist] = useState<Todo[]>([]);
+  const [search, setSearch] = useState<string>("");
+  const [status, setStatus] = useState("all");
+  const [sortBy, setSortBy] = useState<string>("name");
+  const [sortOrder, setSortOrder] = useState<string>("asc");
+
+  const navigate = useNavigate();
+
+  useEffect(() => {
+    if (todos) {
+      setTodolist(todos);
+      // setNextId(todos.length + 1);
+    }
+  }, [todos]);
+  {loading && <div>Loading... </div>}
```

Comment:
This line is inappropriate here move it to start of html code in the file.

Reply:
Done